polaris.shopify.com/content/patterns/card-layout/index.mdx

```diff
@@ -0,0 +1,639 @@
+
+  </Text>
+
+{' '}
+
+<Large>
+  ![A simple schematic showing a stack with two blocks; card header and form
+  layout. The form layout is a stack of form layout items. Both stacks have
+  space-300
+  gaps.](/images/patterns/card-layout/spacing/card-layout-spacing-23-stacks-space-300@2x.png)
+</Large>
+
+  <Medium variant="do">
+    ![An example card showing a form layout with multiple form layout items. The form layout is a stack with space-300 gaps.](/images/patterns/card-layout/spacing/card-layout-spacing-24-stacks-do-use-between-layout-items@2x.png)
+
+    Use space-300 between form layout items.
+
+  </Medium>
+
+  <Medium variant="dont">
+    ![An example card showing a form layout with multiple form layout items. The form layout is a stack with space-200 gaps. The gap size causes unclear separation between two form layout items.](/images/patterns/card-layout/spacing/card-layout-spacing-25-stacks-dont-use-tight-space-between-form-layout-items@2x.png)
+
+    Use too tight spacing between groups of elements that have variable weight and shape, such as form layout items.
+
+  </Medium>
+
+{' '}
+
+<Text>
+  **Space-400** is the loosest card gap and is typically used to space cards
+  with multiple sections. The gap size is the same as the card padding, which
+  structures sections as neatly merged cards.
+</Text>
+
+{' '}
+
+<Large>
+  ![A simple schematic showing a stack with three blocks; card header and two
+  card sections. The stack has space-400 gaps between its blocks. The card
+  sections are stacks with space-200 between its
+  blocks.](/images/patterns/card-layout/spacing/card-layout-spacing-26-stacks-space-400@2x.png)
+</Large>
+
+  <Medium variant="do">
+    ![An example card showing multiple card sections as stacks separated by space-400 gaps. The card sections are stacks with space-200 between its blocks.](/images/patterns/card-layout/spacing/card-layout-spacing-27-stacks-do-use-between-sections@2x.png)
+
+    Use space-400 to separate card sections inside cards.
+
+  </Medium>
+
+  <Medium variant="dont">
+    ![An example card showing multiple card sections as stacks separated by space-400 gaps. The card sections are stacks with space-400 between its blocks, causing too much distance between section titles and section content.](/images/patterns/card-layout/spacing/card-layout-spacing-28-stacks-dont-use-inside-sections@2x.png)
+
+    Use space-400 inside card sections, as it can disconnect content that belongs together. Instead, default to space-200 and upsize to space-300 if the former seems too tight.
+
+  </Medium>
+</LayoutSection>
+
+<LayoutSection card>
+  <Text>
+    ## Alignment
+
+    Content alignment makes cards more balanced and easier for merchants to scan. Content should be vertically aligned along both the left and right edge of the card. It’s always more important to align content [optically](#optically), so that it appears to be aligned rather than technically being aligned.
```

Comment:
@johanstromqvist this was trying to link to a section in the Google doc, but I wasn't sure which section it needs to be linked